### DISCUSSION TITLE
TypeScript - How to properly define defaultValues type

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [airjp73/rvf#210] [Q&A] [answered]
Hi, 
I'm having a hard time trying to properly type the `defaultValues` prop. 

Whatever I try it all fails: 

<img width="276" alt="CleanShot 2022-11-15 at 15 14 11@2x" src="https://user-images.githubusercontent.com/358412/201941501-ab6882ec-ee22-47ab-9cfc-f64805c794ff.png">

I always get the `Type 'XYZ is not assignable to type 'Partial<FormEvent<Element>>'`

What's the correct way to do this?

**Comment by airjp73** [comment, marked as answer]
Hi! Could you show a bit more of the code? The type for default values comes from the validator you pass into the ValidatedForm.

**Reply by piotrkulpinski** [reply]
> The type for default values comes from the validator you pass into the ValidatedForm

That's what I needed, thanks!
